azure-storage/pom.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>spring-cloud-stream-azure-storage-parent</artifactId>
```

Comment:
OK. Looks like we don't understand each other a bit.
Sorry for my bad English or for anything else I'm saying is unclear.
I mean that we should add `<module>azure-storage</module>` into `<modules>` of root pom of the entire project.

I could make the polishing for you do demonstrate what I have in mind on the matter, but I can't do that against your `master`. Having my infrastructure polishing you can integrate that commit into your branch and go ahead with other Blob Storage stuff to reach some finish. But that would be enough complex if you